Sprint-1/exercises/random.js

```diff
@@ -3,7 +3,27 @@ const maximum = 100;
 
 const num = Math.floor(Math.random() * (maximum - minimum + 1)) + minimum;
 
+console.log(num);
+
 // In this exercise, you will need to work out what num represents?
 // Try breaking down the expression and using documentation to explain what it means
 // It will help to think about the order in which expressions are evaluated
 // Try logging the value of num and running the program several times to build an idea of what the program is doing
+
+//------------------------------------------------ANSWER----------------------------------------------------------------------
+
+// 1. Math.random():
+// This part, Math.random(), gives us a random decimal number between 0 and 1. It could be something like 0.234 or 0.857—always between 0 and just under 1.
+
+// 2. Making the random number bigger:
+// We then multiply that random decimal by 100 (since (maximum - minimum + 1) calculates to 100).
+// This turns the decimal into a bigger random number that’s somewhere between 0 and almost 100. For example, it could now be 45.67 or 98.23.
+
+// 3. Rounding down to a whole number:
+// Next, Math.floor(...) rounds down whatever number we have to the nearest whole number. If we had 45.67, it becomes 45. If it was 98.23, it becomes 98.
```

Comment:
Good assessment, but might I suggest that you explain the purpose behind rounding _down_ the value rather than _up_?

Reply:
Thanks for the feedback! So the reason we’re rounding down with Math.floor() is to make sure we stay within the 1 to 100 range. If we rounded up, we’d risk getting a number outside that range—like going over 100. By rounding down and adding + minimum, we keep everything nicely contained.

Hope that clears it up! Let me know if there’s anything else you think I should add. Thanks again!